src/main/java/seedu/application/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,35 @@
+package seedu.application.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.application.model.Model.PREDICATE_SHOW_ALL_APPLICATIONS;
+
+import seedu.application.logic.commands.exceptions.CommandException;
+import seedu.application.model.Model;
+
+/**
+ * Reverts the {@code model}'s application book to its previous state.
+ */
+public class RedoCommand extends Command {
+
+    public static final String COMMAND_WORD = "redo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Restores a previously undone application book state from its history.\n";
+
+    public static final String MESSAGE_SUCCESS = "Redo successfully!";
```

Comment:
Sounds a bit odd to me. How about this instead?
```suggestion
    public static final String MESSAGE_SUCCESS = "Redone successfully!";
```